### DISCUSSION TITLE
Where to find "SQL Server activities" - execute SQL Query

### DISCUSSION THREAD

**Discussion by Nisse123** [elsa-workflows/elsa-core#3848] [Q&A]
I am using the docker image  elsaworkflows/elsa-dashboard-and-server:latest  (2.10.2.2)
I cannot find the "SQL Server activities" mentioned in this issue
https://github.com/elsa-workflows/elsa-core/issues/2947

How do I activate the 'widget' or is there some other way to query a sql database for data?

Thanks,

Screenshot from the issue 2947 displaying the SQL widget
![228238377-6d41b6f5-a127-4dc1-a144-f66143b1b8a8](https://user-images.githubusercontent.com/74415584/228238791-0eadd5ca-2e1a-4372-a801-312917e9d155.png)

**Comment by sfmskywalker** [comment]
Hi @Nisse123 

The Docker image doesn't have the SQL Server activities enabled, and the set of available activities isn't currently configurable from outside the application.

The Docker image is intended as a quick and easy way to take a look at Elsa, and wasn't really designed to be a reusable for anything else.

That being said, I do want to provide a more configurable Docker image for Elsa 3 that offers more control over its configuration.

Until then, I can offer the following suggestions:

1. Create a PR that enables the SQL Activities in the application from which the Docker image is built.
2. Create your own ASP.NET application from which you configure Elsa exactly as you need  (and dockerize as necessary).
